src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/ProviderBase/DbConnectionInternal.cs

```diff
@@ -203,6 +203,14 @@ virtual protected bool ReadyToPrepareTransaction
             }
         }
 
+        virtual internal bool IsAccessTokenExpired
+        {
+            get
+            {
+                return false;
```

Comment:
another question, why are we returning false always? Isn't it safer to consider this expired by default unless it is not?

Reply:
I would see it other way.. token expiry is only specific to federated authentication, otherwise there's no need to doom pooled connection, hence this defaults to false.